src/components/AddRoom/AddRoom.jsx

```diff
@@ -79,7 +79,7 @@ const AddRoom = () => {
 
   return (
     <div>
-      <Title icon="add" header={true}>
+      <Title icon="add" header={true} marginAuto={true}>
```

Comment:
기능상 문제는 없지만 `marginAuto`만 쓰셔도 true로 들어갑니다!
```suggestion
      <Title icon="add" header={true} marginAuto>
```